defaults/main.yml

```diff
@@ -24,6 +24,15 @@ rstudio_workbench_rserver_config:
 rstudio_workbench_rsession_config:
   session-timeout-minutes: 30
 
+############################################################
+# /etc/rstudio/logging.conf
+############################################################
+
+rstudio_workbench_logging_config:
+  '*':
+    log-level: "warn"
```

Comment:
maybe info for lower level? i think info is default for workbench

Reply:
Warn is the default one. I think we can leave it like this. Anyone can change it later. I commented it out so that it will not overwrite default settings provided by rstudio.